DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_OverviewTable.cshtml

```diff
@@ -9,7 +9,7 @@
       </th>
       <th class="" scope="col">
         @(string.IsNullOrWhiteSpace(Model.FirstOrDefault().QuestionLabel) ?
-            "Question" : Model.FirstOrDefault().QuestionLabel)
+            "Assessment" : Model.FirstOrDefault().QuestionLabel)

```

Comment:
@OluwatobiAwe, I think this text was correct. It is the mobile only label (`<span class="nhsuk-table-responsive__heading">`) in the `_AssessmentQuestionOverviewCells` partial below that needs altering to match this.

Reply:
@kevwhitt-hee , I've reverted to **@(string.IsNullOrWhiteSpace(Model.FirstOrDefault().QuestionLabel) ?
            "Question" : Model.FirstOrDefault().QuestionLabel)**, then passed the result of the evaluation to the _AssessmentQuestionOverviewCells partial view for consistency.